docs/user/gcp/install_upi.md

```diff
@@ -321,6 +321,7 @@ Export variables needed by the resource definition.
 ```sh
 export CONTROL_SUBNET=`gcloud compute networks subnets describe ${INFRA_ID}-master-subnet --region=${REGION} --format json | jq -r .selfLink`
 export CLUSTER_IMAGE=`gcloud compute images describe ${INFRA_ID}-rhcos-image --format json | jq -r .selfLink`
+export ZONES=(`gcloud compute regions describe ${REGION} --format=json | jq -r .zones | tr "\"" "/" | grep -v "\[" | grep -v "\]" | cut -d "/" -f10`)
```

Comment:
The post-`jq` stuff seems complicated.  And you're building a Bash array?  I think arrays are a non-POSIX Bashism.  Maybe tgat's ok, or maybe we should use `ZONE_1`, etc. holding individual values?

Reply:
Correct, it was producing an array. Switched to using ZONE_0, ZONE_1, and ZONE_2. As a result, the post-jq bits are simplified.